integration-tests/cross-contract-calls/e2e_tests.rs

```diff
@@ -0,0 +1,84 @@
+use super::cross_contract_calls::*;
+use ink_e2e::ContractsBackend;
+
+type E2EResult<T> = std::result::Result<T, Box<dyn std::error::Error>>;
+
+#[ink_e2e::test]
+async fn flip_and_get<Client: E2EBackend>(mut client: Client) -> E2EResult<()> {
+    // given
+    let other_contract_code = client
+        .upload("other-contract", &ink_e2e::alice())
+        .submit()
+        .await
+        .expect("other_contract upload failed");
+
+    let mut constructor = CrossContractCallsRef::new(other_contract_code.code_hash);
+    let contract = client
+        .instantiate("cross-contract-calls", &ink_e2e::alice(), &mut constructor)
+        .submit()
+        .await
+        .expect("basic-contract-caller instantiate failed");
+    let mut call_builder = contract.call_builder::<CrossContractCalls>();
+    let call = call_builder.flip_and_get_v1();
+
+    // when
+    let result = client
+        .call(&ink_e2e::alice(), &call)
```

Comment:
Can we get the actual ref time and proof time used for the call to test against?

Reply:
Please clarify what you mean

Reply:
In cargo contract, when executing a call, you get the response of how much gas (ref and proof time) was actually used. Can we use that info in e2e test as well?

Reply:
The e2e `call` already does this transparently by doing a dry run first and passing the estimated limits to the actual tx.

Do you mean for the other test which passes explicit limits for the sub call? So dry run the sub call and use those values instead of the constants? We could do that, what would be the benefit though?

Reply:
> We could do that, what would be the benefit though?

Purely for illustrative purposes so we test that ref and proof times are set and estimated correctly.